renderer/sources.gni

```diff
@@ -4,10 +4,21 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 brave_chrome_renderer_sources = [
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_render_frame_observer.cc",
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_render_frame_observer.h",
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_handler.cc",
```

Comment:
needs `gn format` (though CI shouldn't allow merging until all formatting is correct)

Reply:
ran `gn format <file_name>` on all gn files I've touched and committed as a separate commit